yapapi/engine.py

```diff
@@ -768,13 +772,12 @@ async def reject_proposal(reason: str) -> events.ProposalRejected:
                 # reject proposal if there are no common payment platforms
                 return await reject_proposal("No common payment platform")
 
-            # Check if the timeout for debit note acceptance is not too low
-            timeout = proposal.props.get(DEBIT_NOTE_ACCEPTANCE_TIMEOUT_PROP)
-            if timeout:
-                if timeout < DEBIT_NOTE_MIN_TIMEOUT:
-                    return await reject_proposal("Debit note acceptance timeout is too short")
-                else:
-                    demand_builder.properties[DEBIT_NOTE_ACCEPTANCE_TIMEOUT_PROP] = timeout
+            try:
+                demand_builder = await self.engine._strategy.answer_to_provider_offer(
+                    demand_builder, proposal
+                )
+            except Exception as e:
```

Comment:
If we raise only ValueError on purpose (check other comments), here we'd like also to capture only ValueError.